app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<p style="color: green"><%= notice %></p>
+
+
+<div class="flex-center-col recipes-container">
+  <div style="position: relative;">
+    <h1>Recipes</h1>
+    <% if @user === current_user %>
+      <%= link_to "Add recipe", new_user_recipe_path(user_id: @user.id), class: "add-recipe btn btn-success" %>
+    <%end%>
+  </div>
+
+  <div id="recipes">
+    <% @recipes.each do |recipe| %>
+      <%= render partial: 'recipe_card', locals: { recipe: } %>
+    <% end %>
+  </div>
+  <% if @recipes.empty? %>
+     <h2>No Recipes Added</h2>
+    <%end%>
+</div>
```

Comment:
- [Optional] Right now I'm getting the error in the short video below when I try to click on a recipe item. It will be nice to fix it.

[Screencast from 06-30-2023 03:51:34 PM.webm](https://github.com/c00p75/recipe-app/assets/63087491/d30fec48-36b7-4260-95ea-07338dd7fc1b)

Reply:
Ok, yaa that is because of Bullet, N+1 query. Sure we will fix it.
Thanks a lot